server/container/cli/src/main/java/org/apache/james/cli/ServerCmd.java

```diff
@@ -102,7 +105,7 @@ public static void main(String[] args) {
         } catch (IOException ioe) {
             failWithMessage("Error connecting to remote JMX agent : " + ioe.getMessage());
         } catch (Exception e) {
-            e.printStackTrace();
+            LOG.error("Error on command: ", e);
             failWithMessage("Error while executing command:" + e.getMessage());
```

Comment:
I'm not asking for removing the log, I'm asking for making the message returned to the user a bit more specific.

About empty messages, maybe you did not encountered it, but I did ;-).